shell/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -84,6 +91,10 @@ public final void before() throws Exception {
     mNewOutput = new PrintStream(mOutput);
     mOldOutput = System.out;
     System.setOut(mNewOutput);
+    // clear the loginUser
+    Field field = LoginUser.class.getDeclaredField("sLoginUser");
```

Comment:
Could this be used for setting a private static field? Thanks.

I take a try like this `Whitebox.setInternalState(LoginUser.class, "sLoginUser", null)`, but got FieldNotFoundException. I guess it is a mis-use?